bug: panic in nettagadd

```bash
$ sudo wwctl node set --netname second --nettagadd vlan=192 c1
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/pkg/node.(*NodeConf).getterFrom(0x4000225dc0, {{{0x4000382500, 0x1, 0x1}, {0x0, 0x0, 0x0}, {0x0, 0x0}, {0x0, ...}}, ...}, ...)
	/home/janderson/rpmbuild/BUILD/warewulf-4.3.0/internal/pkg/node/transformers.go:179 +0x20b0
github.com/hpcng/warewulf/internal/pkg/node.(*NodeConf).GetRealFrom(...)
	/home/janderson/rpmbuild/BUILD/warewulf-4.3.0/internal/pkg/node/transformers.go:17
github.com/hpcng/warewulf/internal/pkg/node.(*NodeYaml).NodeUpdate(0x400061fc48, {{{0x4000382500, 0x1, 0x1}, {0x0, 0x0, 0x0}, {0x0, 0x0}, {0x0, ...}}, ...})
	/home/janderson/rpmbuild/BUILD/warewulf-4.3.0/internal/pkg/node/modifiers.go:58 +0x1fc
github.com/hpcng/warewulf/internal/pkg/api/node.NodeSetParameterCheck(0x400061fcc0, 0x0)
	/home/janderson/rpmbuild/BUILD/warewulf-4.3.0/internal/pkg/api/node/node.go:263 +0x940
github.com/hpcng/warewulf/internal/app/wwctl/node/set.CobraRunE(0x145a900, {0x40003c29b0, 0x1, 0x5})
	/home/janderson/rpmbuild/BUILD/warewulf-4.3.0/internal/app/wwctl/node/set/main.go:40 +0x300
github.com/spf13/cobra.(*Command).execute(0x145a900, {0x40003c2960, 0x5, 0x5})
	/home/janderson/rpmbuild/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:850 +0x678
github.com/spf13/cobra.(*Command).ExecuteC(0x1456220)
	/home/janderson/rpmbuild/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:958 +0x418
github.com/spf13/cobra.(*Command).Execute(...)
	/home/janderson/rpmbuild/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:895
main.main()
	/home/janderson/rpmbuild/BUILD/warewulf-4.3.0/cmd/wwctl/main.go:13 +0x34
```

Node in question:

```
$ sudo wwctl node list -a c1
NODE                 FIELD              PROFILE      VALUE
=====================================================================================
c1                   Id                 --           c1
c1                   comment            default      This profile is automatically included for each node
c1                   cluster            --           --
c1                   container          --           --
c1                   ipxe               --           (default)
c1                   runtime            --           (generic)
c1                   wwinit             --           (wwinit)
c1                   root               --           (initramfs)
c1                   discoverable       --           --
c1                   init               --           (/sbin/init)
c1                   asset              --           --
c1                   kerneloverride     --           --
c1                   kernelargs         --           (quiet crashkernel=no vga=791)
c1                   ipmiaddr           --           --
c1                   ipminetmask        --           --
c1                   ipmiport           --           --
c1                   ipmigateway        --           --
c1                   ipmiuser           --           --
c1                   ipmipass           --           --
c1                   ipmiinterface      --           --
c1                   ipmiwrite          --           --
c1                   profile            --           default
c1                   default:type       --           --
c1                   default:onboot     --           --
c1                   default:netdev     --           eth0
c1                   default:hwaddr     --           --
c1                   default:ipaddr     --           10.0.0.100
c1                   default:ipaddr6    --           --
c1                   default:netmask    --           --
c1                   default:gateway    --           --
c1                   default:primary    --           true
c1                   second:type        --           --
c1                   second:onboot      --           --
c1                   second:netdev      --           eth0.192
c1                   second:hwaddr      --           --
c1                   second:ipaddr      --           10.0.0.101
c1                   second:ipaddr6     --           --
c1                   second:netmask     --           --
c1                   second:gateway     --           --
c1                   second:primary     --           true
```

--- a/internal/pkg/api/node/list.go
+++ b/internal/pkg/api/node/list.go
@@ -116,7 +116,11 @@ func NodeList(nodeGet *wwapiv1.GetNodeList) (nodeList wwapiv1.NodeList, err erro
 						for j := 0; j < netInfoType.NumField(); j++ {
 							netConfField, ok := netConfType.FieldByName(netInfoType.Field(j).Name)
 							if ok {
-								fieldName = netName + ":" + netConfField.Tag.Get("lopt")
+								if netConfField.Tag.Get("lopt") != "nettagadd" {
+									fieldName = netName + ":" + netConfField.Tag.Get("lopt")
+								} else {
+									fieldName = netName + ":tag"
+								}
 							} else {
 								fieldName = netName + ":" + netInfoType.Field(j).Name
 							}

--- a/internal/pkg/node/transformers.go
+++ b/internal/pkg/node/transformers.go
@@ -164,6 +164,10 @@ func (nodeConf *NodeConf) getterFrom(nodeInfo NodeInfo,
 							} else if netVal.Type() == reflect.TypeOf(map[string]*Entry{}) {
 								entryMap := netVal.Interface().(map[string](*Entry))
 								confMap := netConfVal.Elem().Field(j).Interface().(map[string]string)
+								if confMap == nil {
+									confMapPtr := netConfVal.Elem().Field(j).Addr().Interface().(*map[string]string)
+									*confMapPtr = make(map[string]string)
+								}
 								if len(confMap) > len(entryMap) {
 									for confMapKey := range confMap {
 										foundKey := false

